psutil/_pslinux.py

```diff
@@ -1243,7 +1243,9 @@ def multi_cat(*paths):
     energy_full = multi_cat(
         root + "/energy_full",
         root + "/charge_full")
-    if energy_now is None or power_now is None:
+    if power_now is None: 
+        power_now = 0 # provide a fallback since some systems don't provide this.
```

Comment:
This doesn't look right. If `power_now` can't be determined we also cannot determine `secsleft` so it's probably better to just return None

Reply:
but in case `power_now` is not available this still gives the battery percentage and whether or not the charger is connected